src/anshuKJun20/IntersectionOfArray.java

```diff
@@ -0,0 +1,44 @@
+package anshuKJun20;
+
+import java.util.Arrays;
+
+public class IntersectionOfArray {
+	
+	double[] getIntersectionArray(double[] arr1, double[] arr2) {
+		double[] tempArr = new double[arr2.length];
+		int count = 0;
+		for (int outerindex = 0; outerindex < arr2.length; outerindex++) {
+			boolean flag = false;
+			for (int innerindex = 0; innerindex < arr1.length; innerindex++) {
+				if (arr2[outerindex] == arr1[innerindex]) {
+					flag = true;
+					break;
+				}
+			}
+			if (flag == true) {
```

Comment:
1.First if statement you are changing flag value to 'true' and again you are checking for 
if (flag == true) condition.This is not correct.
2.Without setting flag value also you can get tempArr[] .

Reply:
You are right, No need of flag here. Changes Done.